assets/javascripts/lib/discourse.js

```diff
@@ -74,6 +74,15 @@ class TopicTitle {
   }
 }
 
+/**
+ * Resets plugin state
+ *
+ * Used in tests.
+ */
+export function _reset() {
```

Comment:
As a general rule, if we export something it's not private and should not begin with an underscore. I also recommend changing it to `resetUserIdentity` so that it's more clear in the file that imports it what `reset` is doing when called.

Reply:
This function is strange because it is part of the private API, but it must be exported. It is intended to be used only in tests. I thought we use the same convention in core, but looking again now, I can only see `export function _clearSnapshots`.

Reply:
I implemented your suggestion to throw an error if called outside tests. It's great, thank you! :+1: